core/src/test/scala/integration/kafka/api/AbstractConsumerTest.scala

```diff
@@ -158,7 +158,7 @@ abstract class AbstractConsumerTest extends BaseRequestTest {
       records ++= polledRecords.asScala
       records.size >= numRecords
     }
-    TestUtils.pollRecordsUntilTrue(consumer, pollAction, waitTimeMs = 60000,
+    TestUtils.pollRecordsUntilTrue(consumer, pollAction(_), waitTimeMs = 60000,
```

Comment:
I have changed the name of method to make the build fix here: https://github.com/confluentinc/kafka/pull/1121. As I didn't see `(_)` this being used elsewhere in Kafka other than 1 instance.

Reply:
thanks for fixing the build, I'll revert my changes for the build fix